routes/api/stream/index.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
```

Comment:
Add a third param called `next` here, which can be used to pass errors down the first error handler it finds down the chain.

See: https://expressjs.com/en/guide/error-handling.html